src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/limitedpartner/dto/LimitedPartnerDataDto.java

```diff
@@ -22,7 +22,6 @@ public class LimitedPartnerDataDto extends PartnerDataDto {
     private Currency contributionCurrencyType;
 
     @JsonProperty(CONTRIBUTION_CURRENCY_VALUE_FIELD)
-    @Pattern(regexp = "^\\d+(\\.\\d{2})$", message = "Value must be a valid decimal number")
```

Comment:
Not sure why this has gone. Format validation still applies, the same as in the web.

Reply:
Ok put the regex back.